### DISCUSSION TITLE
item-map-downloader все карты сохраняются перевернутыми и в разном порядке

### DISCUSSION THREAD

**Discussion by wristikl** [PrismarineJS/mineflayer#3410] [Q&A]
item-map-downloader все карты сохраняются перевернутыми и в разном порядке
должно быть так 
![image](https://github.com/PrismarineJS/mineflayer/assets/167366943/1612c96d-84cc-491d-8e6e-095f15977e3d)
а у меня вот так 
![map_000008](https://github.com/PrismarineJS/mineflayer/assets/167366943/aea0cead-515f-4feb-bf1f-25d80dd63457)![map_000040](https://github.com/PrismarineJS/mineflayer/assets/167366943/9fa12fb6-04e3-497d-9422-d91331a40829)![map_000051](https://github.com/PrismarineJS/mineflayer/assets/167366943/0dac3fca-6194-4b4e-8000-6d88e09d8c29)
![map_000081](https://github.com/PrismarineJS/mineflayer/assets/167366943/662b4572-6787-4944-a530-d4e21adc53c7)![map_000092](https://github.com/PrismarineJS/mineflayer/assets/167366943/81f5cdb8-4a6c-41d1-8f18-bfd4afd8de61)![map_000122](https://github.com/PrismarineJS/mineflayer/assets/167366943/0104e4d3-4741-42af-8e00-8a7edd1d8204)

**Comment by greenik228** [comment]
Для начала, советую общаться на английском языке, чтобы другие участники могли понять вас:)
Логичным вариантом был бы подбор всех возможных вариантов, т.е. позиции и поворота каждой части картинки. Но я всё ещё думаю, как программа будет находить ту самую нужную картинку.

To begin with, I advise you to communicate in English so that other participants can understand you:)
A logical option would be to select all possible options, i.e. position and rotation of each part of the picture. But I’m still wondering how the program will find the very desired picture.
